### PR TITLE
v1.3.0 version update.

### PR DESCRIPTION
**Pull request by dzimine**

### CODE REVIEW

**Comment by Kami** [comment]
Alright everyone, I reset this branch and the puppet-st2 one.

I reset it to the originally commit and merged inthe installer change (convergence fix). This branch can again be used to test v1.3.0 stuff.

**Comment by Kami** [comment]
The installer stuff doesn't seem to work, I suspect one of the reason for that might be #331, but I'm not 100% sure and I could be wrong.

In any case, I'm honestly getting sick of this mess.  I spent most of last two weeks dealing with the current mess - debugging and fixing installer issues (and new issues keep popping up all the time).

The problem is that there are simply too many moving pieces and puppet is simply too hard to debug and reason about. The whole develop and test a change cycle is simply way too slow and manual. And debugging and fixing issues at this point is more magic than science. We can't figure out root causes so we are simply adding on hack / work around on top of another.

In short what we have is not maintainable in any reasonable manner (and time frame (and we are not investing enough into automated tests for the current state of the arts) so we need to come up with a better short term plan (imo, we simply can't wait until new packages are out and ready since this is too far out).

**Comment by manasdk** [comment]
Going to merge this into master to unleash v1.3.0 onto the world!
